Package.swift

```diff
@@ -1542,9 +1558,13 @@ func firestoreTargets() -> [Target] {
         "FirebaseCoreExtension",
         "leveldb",
         "FirebaseSharedSwift",
+        "FirebaseFirestoreCpp",
       ],
       path: "Firestore/Swift/Source",
       resources: [.process("Resources/PrivacyInfo.xcprivacy")],
+      swiftSettings: [
+        .interoperabilityMode(.Cxx), // C++ interoperability setting
+      ],
```

Comment:
Maybe try commenting this out since the the viral-ness may be circumvented  by the implementation-only import.
```suggestion
//      swiftSettings: [
//        .interoperabilityMode(.Cxx), // C++ interoperability setting
//      ],
```